src/main.js

```diff
@@ -87,8 +98,14 @@ function displayUserBook() {
   grabTagline1.innerText = currentCover[0].tagline1;
   grabTagline2.innerText = currentCover[0].tagline2;
 }
+function saveCurrentCover (){
+  savedCovers.push(currentCover[0]);
+  currentCover.shift()
+  console.log(savedCovers, 'savedCovers');
+  console.log(currentCover, `currentCover`);
+}
```

Comment:
This is great. Glad we solved the issue of seamlessly transferring objects from arrays.